src/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixer.php

```diff
@@ -169,13 +163,8 @@ protected function applyChainedCallsFix(Tokens $tokens)
 
     /**
      * Find the index for the new line. Return the given index when there's no new line.
-     *
-     * @param int    $index
-     * @param Tokens $tokens
-     *
-     * @return int
      */
-    private function getNewLineIndex($index, Tokens $tokens)
+    private function getNewLineIndex(int $index, Tokens $tokens): int
```

Comment:
I'd love this, but I think PHP CS Fixer doesn't allow this yet, since it support PHP 5.6